### DISCUSSION TITLE
How much blood remains in organs after blood sampling via cardiac puncture?

### DISCUSSION THREAD

**Discussion by wangwei1619** [Open-Systems-Pharmacology/Forum#1294] [Questions & Problems]
Hi, all,

I am wondering how much blood remains in organs after blood sampling via cardiac puncture. I have searched that nearly half of all blood can be sampled via cardiac puncture. So where is the remaining blood mainly staying, venous vein, organs, or both?

Any comment is appreciated

Wang Wei

**Comment by JanSchlender** [comment]
Hi Wang,
Tricky question. This will most likely also depent on the species. Since the heart of the mice is too small for this procedure, you will most likely obtain mixed blood (wrt oxygene saturation). For larger species, the amount of blood exeeding the cardiac blood volume will most likely be removed from the venous blood pool. In this case I would keep the pulmonary blood content, arterial blood pool and the vascular space of each organ but substract the venous blood pool as far as needed. Does the sampling volume exeed the venous blood pool in your case?
Please keep in mind, that the vascular space of the heart in PK-Sim represents the blood volume of the myocadial tissue. The PBPK framework has no representation of a ventricle. 
Best,
Jan

**Reply by wangwei1619** [reply]
Hi, Jan,

Thanks for your reply. In fact, I am trying to build a model for data from a paper, where the authors collected **mice** blood via cardiac puncture and estimated the proportion of the initial dose remaining in the blood pool at various time points.  They also obtained organs and estimated the dose remaining there.  I think it is reasonable to consider they have sampled blood as much as they can, so the question is where and how much the remaining blood is since this is related to organ drug concentration calculation.

**Comment by Christoph27** [comment]
Hi Wang,
we estimated a fraction of 18% of residual blood after tissue dissection assuming the same fraction for all organs (https://doi.org/10.1007/s10928-017-9559-4).
In a more recent paper, they looked at residual blood organ specific: https://doi.org/10.1113/JP274819 

These numbers are however not specific for a condition after cardiac puncture.

Best,
Christoph

**Reply by wangwei1619** [reply]
Hi, Christoph

Thanks a lot. I think it is also important information. I understand that 18% of organ blood (not total blood) still stays in organs after dissection, do I recognize it rightly?  

Best,
Wang Wei

**Reply by Christoph27** [reply]
Correct! The parameter "fraction of blood for sampling" used in the "Whole Organ" Observer the would be 0.18.  
It's different for the information in  https://doi.org/10.1113/JP274819 which you have to convert.

**Reply by wangwei1619** [reply]
The residual plasma fraction in https://doi.org/10.1113/JP274819 is plainly plasma fractions to organ volume, right? 
As for the "fraction of blood for sampling" used in the "Whole Organ" Observer, I do not remember where to find it in PK-Sim.

**Reply by Christoph27** [reply]
yes, that is also my understanding: what they report in  https://doi.org/10.1113/JP274819 as "residual plasma fraction" is residual plasma volume per tissue weight, unit ml/g (i.e. "to organ volume" if one assumes a tissue density of 1 g/ml).  
The 18% in https://doi.org/10.1007/s10928-017-9559-4 (="fraction of blood for sampling" 0.18 in PK-Sim) is in contrast the fraction with respect to the total blood volume assigned to the organ in PK-Sim for the in vivo situation), i.e. 
fraction = V_residual after tissue dissection / V_in vivo. 
You can find that parameter in PK-Sim at the individual at "Sampling" (see screenshot below) or similarly at the simulation.

![fr](https://user-images.githubusercontent.com/26439150/187862651-24482544-f6fc-4c78-912f-b0fc8e1999d7.jpg)

**Reply by wangwei1619** [reply]
Thanks a lot. It is very useful.
